src/fill/alloy.rs

```diff
@@ -20,29 +27,31 @@ impl Tx for Signed<alloy::consensus::TxLegacy> {
             max_fee_per_blob_gas,
             authorization_list,
         } = tx_env;
+        *tx_type = TxType::Legacy as u8;
         *caller = self.recover_signer().unwrap();
         *gas_limit = self.tx().gas_limit;
-        *gas_price = U256::from(self.tx().gas_price);
-        *transact_to = self.tx().to;
+        *gas_price = self.tx().gas_price;
+        *kind = self.tx().to;
         *value = self.tx().value;
         *data = self.tx().input.clone();
-        *nonce = Some(self.tx().nonce);
+        *nonce = self.tx().nonce;
         *chain_id = self.tx().chain_id;
-        access_list.clear();
+        access_list.0.clear();
         gas_priority_fee.take();
         blob_hashes.clear();
-        max_fee_per_blob_gas.take();
-        authorization_list.take();
+        *max_fee_per_blob_gas = 0;
```

Comment:
should this be 0?

Reply:
so it now must be populated (no longer an option type), but is ignored during tx validation for non-4844 tx types

https://github.com/bluealloy/revm/blob/6f383227cad2c0eb4949533a2e27de015ca6bdb9/crates/handler/src/validation.rs#L102

Reply:
so i think any value is permissible